src/EditorFeatures/VisualBasic/LineCommit/CommitBufferManager.vb

```diff
@@ -156,10 +141,33 @@ Namespace Microsoft.CodeAnalysis.Editor.VisualBasic.LineCommit
         End Sub
 
         Private Structure FormattingInfo
+            ' This is the original flag to enable semantic formating on a span
```

Comment:
I think doc comments makes more sense here.

Reply:
Fixed, added suppressions on Name Syntax Rule and replaced comments with Doc Comments.

Reply:
done